.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "rust-analyzer.cargo.allFeatures": true
+}
```

Comment:
Are you sure you want to commit this?

Reply:
good question, without it I would not get warnings/suggestions for the webserver part of the json-rpc crate, but 2 reasons against it:
1. we might have mutually exclusive features
2. most of the webserver code is now in yerpc-tide, so we might not need the linting so much anyway.

maybe we can also enable just the web-server feature instead of all features? IDK what vscode supports